packages/x-tree-view/src/internals/plugins/useTreeViewKeyboardNavigation/useTreeViewKeyboardNavigation.ts

```diff
@@ -127,7 +127,11 @@ export const useTreeViewKeyboardNavigation: TreeViewPlugin<
       }
 
       // If the tree is empty there will be no focused node
-      if (event.altKey || event.currentTarget !== event.target || state.focusedNodeId == null) {
+      if (
+        event.altKey ||
+        event.currentTarget !== event.target ||
+        models.focusedNode.value == null
```

Comment:
Given how often we access `models.focusedNode.value` which is quite verbose, WDYT about creating a variable?

```ts
const focusedNode = models.focusedNode.value;
```